### DISCUSSION TITLE
Gameplan Show Whitescreen

### DISCUSSION THREAD

**Discussion by denzizzy966** [frappe/gameplan#273] [General]
i had to install frappe and gameplan on my vps. but why gameplan show whitescreen only ? can anyone show the proper install for gameplan and erpnext v15 in one server?

**Comment by Jalpara** [comment]
+1

**Reply by ukwuezeobinna** [reply]
I just installed GamePlan on Frappe v15 and i dont know where to start from. Is there a home page or something?

**Reply by denzizzy966** [reply]
site_url/g/home
